Formula/gdu.rb

```diff
@@ -16,7 +16,12 @@ class Gdu < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "build", *std_go_args, "-ldflags", "-s -w -X main.AppVersion=v#{version}"
+    time = Time.new
+    user = `id -u -n`
+    system "go", "build", *std_go_args, "-ldflags", "-s -w \
+      -X 'github.com/dundee/gdu/build.Version=v#{version}' \
+      -X 'github.com/dundee/gdu/build.Time=#{time}' \
+      -X 'github.com/dundee/gdu/build.User=#{user}'"
```

Comment:
I think this might work better, and is more consistent with other formulae:
```suggestion
    user = `Utils.safe_popen_read("id", "-u", "-n")
    
    ldflags = %W[
      -s -w
      -X github.com/dundee/gdu/build.Version=v#{version}
      -X github.com/dundee/gdu/build.Time=#{time}
      -X github.com/dundee/gdu/build.User=#{user}
    ]
    
    system "go", "build", *std_go_args, "-ldflags", ldflags.join(" ")
```
Please try it out locally; I may have may a syntax error somewhere there.

Reply:
Will do. Thank you!